tests/scripts/pulp_2to3_migration/test_plan.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "pulp_2to3_migration" || exit 3
```

Comment:
Am i right, that these tests will not run in our container based CI?
Would this be reason to have this plugin packaged independently?

Reply:
> Am i right, that these tests will not run in our container based CI?

That's correct

> Would this be reason to have this plugin packaged independently?

That's a good question. Let's discuss at our CLI meeting?